commands/run/config/run_profile.ini

```diff
@@ -0,0 +1,61 @@
+[wgs-single]
```

Comment:
The entry point to the runner, the specified "run mode". Here is determined both what pipeline to use, what samples to use and what csv_template.

csv_template needs to be here to account for single <-> trios and similar.